jenkins/control-slave.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
+DOCKER_VERSION='1.9.1-0~wheezy'
+GO_VERSION='go1.6.linux-amd64.tar.gz'
+TIMEZONE='America/Los_Angeles'
+MASTER='jenkins-master'
+
+
+DISK_SIZE='500GB'  # TODO(fejta): 300G for local-ssd
+DISK_TYPE='pd-ssd'   # TODO(fejta): local-ssd
+IMAGE='debian-7-backports'
+MACHINE_TYPE='n1-highmem-32'  # TODO(fejta): optimize to lower values
+SCOPES='cloud-platform,compute-rw,storage-full'  # TODO(fejta): verify
+
+if [[ -z "${1:-}" ]]; then
+  echo "Usage: $(basename "${0}") --small <instance> [AUTO|create|delete|update|reboot|attach]"
+  exit 1
+fi
+
+if [[ "${1:-}" == "--small" ]]; then
+  shift
+  echo 'Reducing resource footprint for script testing...'
+  DISK_SIZE='200GB'
+  DISK_TYPE='pd-standard'
+  MACHINE_TYPE='n1-standard-1'
+fi
+
+INSTANCE="${1}"
+CMD="${2:-auto}"
+
+auto-slave() {
```

Comment:
Jenkins started calling these "agents" in 2.0, so we may want to update our language.